configs/login_server/grd_login.properties

```diff
@@ -22,6 +22,8 @@ loginServer.db.user = root
 loginServer.db.password = 
 loginServer.db.port = 3306
 
+frontend.checkEmailPath = http://localhost/account/checkEmail 
+
 email.disable = true

```

Comment:
```suggestion
loginServer.db.port = 3306

frontend.checkEmailPath = http://localhost/account/checkEmail

email.disable = true

```

Reply:
I believe those characters are windows lineendings. Common standard is linux line endings on the remote.
![image](https://user-images.githubusercontent.com/1238238/112560526-b76e3100-8dd3-11eb-966d-e7268b824e05.png)

Reply:
https://github.com/atom/atom/issues/3105